drivers/vmwarevcloudair/vcloudair.go

```diff
@@ -407,66 +411,17 @@ func (d *Driver) Create() error {
 		return err
 	}
 
-	if d.Provision {
-		dockerInstall := "curl -sSL https://get.docker.com/ | sudo sh"
-
-		log.Infof("Installing Docker...")
-
-		cmd, err = d.GetSSHCommand(dockerInstall)
-		if err != nil {
-			return err
-		}
-
-		if err = cmd.Run(); err != nil {
-			return err
-		}
+	dockerInstall := "curl -sSL https://get.docker.com/ | sudo sh"
```

Comment:
Right, that's exactly my point.  The scripts at /ubuntu/ and / are very,
very different, and the one at /ubuntu/ is meant to be a reference, not
something users actually run (and how that crept into the docs is beyond my
understanding -- it makes me want to turn the script into HTML explicitly
so people stop that).